Added back button to interfaceInfo.vue

Alopalao opened the pull request:
Base on issue #17 
This pull request depends on changes to [k-info-panel/switch_info.kytos](https://github.com/kytos-ng/topology/blob/master/ui/k-info-panel/switch_info.kytos).
Let me know what do you think about this approach.

Alopalao commented on src/kytos/interfaceInfo.vue:
```diff
@@ -88,6 +88,14 @@ export default {
+                           icon: "gear",
+                           title: "Switch Details",
+                           subtitle: this.content["content_switch"].connection,}
+      this.$kytos.$emit("showInfoPanel", panel_content)
```
FIxed with on two commits from UI [e596f70691a94afa02bf0808a48f45ed74e20276 ](https://github.com/kytos-ng/ui/pull/24/commits/e596f70691a94afa02bf0808a48f45ed74e20276) and from topology [707358efeb6cba6e5e8365bcc74257b6b2386bbc](https://github.com/kytos-ng/topology/pull/86/commits/707358efeb6cba6e5e8365bcc74257b6b2386bbc)

viniarck commented:
@Alopalao the latest commits you've pushed look great to me, great fix. Could you merge `master` into this branch again? Please, update the changelog accordingly again, and also set a new beta version `2022.2.0.b1` (b1 this time)

Alopalao commented:
Updated changelog.rst on commit [bd0088895027aab34774b6a6f83aeece54478705](https://github.com/kytos-ng/ui/pull/24/commits/bd0088895027aab34774b6a6f83aeece54478705) and fixed date on commit [67983f1c6b4bf63eb051ea33e3d6cf67ff038691](https://github.com/kytos-ng/ui/pull/24/commits/67983f1c6b4bf63eb051ea33e3d6cf67ff038691)
